src/model_backend/model_api.cpp

```diff
@@ -1,8 +1,9 @@
 #include "model_api.hpp"
 #include "logging.hpp"
+#include "transcribe_service.hpp"
 
 // Function to execute a command and get the output
-std::string model_api::exec_command(const std::string &command)
+std::string ModelApi::ExecCommand(const std::string &command)
```

Comment:
Variables and functions start with a lowercase letter

This is from the README, please change all such variable and function names to start with a lowercase letter. Only classes and structs start with uppercase